test/tests/ver-1e/comparison_ver-1e.py

```diff
@@ -4,28 +4,50 @@
 from matplotlib import gridspec
 import pandas as pd
 from scipy import special
+from numpy import sin,tan,sqrt,exp
 
 # ========= Comparison of concentration as a function of time ===================
 
 fig = plt.figure(figsize=[6.5, 5.5])
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./ver-1e_csv.csv")
+tmap_sol = pd.read_csv("./gold/ver-1e_csv.csv")
 tmap_time = tmap_sol['time']
 tmap_conc = tmap_sol['conc_point1']
 
-analytical_sol = pd.read_csv("./analytical_time.csv")
-analytical_time = analytical_sol['t']
-analytical_conc = analytical_sol['u']
+#Analytical solution
+t = np.expand_dims(tmap_time,axis=0)
+x = 15.75e-6
+c0 = 50.7079            # concentration at the PyC free surface (moles/um^3)
+a  = 33e-6              # thickness of the PyC layer (um)
+l  = 66e-6              # thickness of the SiC layer (um)
+D_PyC = 1.274e-7        # diffusivity in PyC (m^2/s)
+D_SiC = 2.622e-11       # diffusivity in SiC (m^2/s)
+
+k = sqrt(D_PyC/D_SiC)
+
+#Calculate lambda values for analytical solution
+lambda_range = np.arange(1e-12,1e5,1e-4)
+f = 1/tan(lambda_range*a)
+g = 1/tan(k*l*lambda_range)/k
+idx = np.where(np.diff(np.sign(f+g)))
+lambdas = np.expand_dims(lambda_range[idx][::2],axis=0)
+
+summation = (sin(a*lambdas)*sin(k*l*lambdas)*sin(k*(l-x)*lambdas)/(lambdas*( a*sin(k*l*lambdas)*sin(k*l*lambdas) + l*sin(a*lambdas)*sin(a*lambdas) ) ) )*exp(-D_PyC*np.power(lambdas,2)*t.transpose() )
+sums = np.sum(summation,axis=1)
+
+analytical_conc = c0*(D_PyC*(l-x)/(l*D_PyC + a*D_SiC)  - 2*sums)*np.ones(sums.shape)
```

Comment:
I know this is not what's done in the TMAP4 and TMAP7 reports but I would suggest comparing the computed solution to the analytical solution in several x positions, not just one far away from the interface.

Before this fix, the analytical solution and the numerical solution agreed for one position but not for the others.

The comparison plot could look like this:

![image](https://github.com/idaholab/TMAP8/assets/40028739/b544e55d-ff99-4af4-902c-e09528ff13dc)

Reply:
That's a good point, @RemDelaporteMathurin. @ChaitanyaBhave-e, could you add a few more spots (in addition to the TMAP4 and TMAP7 spots)? 

I also want to point out that because we have tests with the exodus files generated by TMAP8, the tests already track the position at each point of he mesh and make sure that new changes do not affect that. Coupled with comparing against the analytical solution in a handful of points, we're well covered.

Reply:
One point here, TMAP4 and 7 do not provide an analytical solution for the IPyC side of the interface. I could try to find the original derivations from Carslaw and Jaeger, which should have solutions for both sides. But for now even if we plot a few more points it's not a proper analytical solution verification - it's equivalent to the exodiff we are already performing. Another option would potentially be to autogenerate a gif from the simulation output and add it to the site documentation.

Reply:
Oh I didn't know this my bad. It's true that we only compared FESTIM to TMAP8 here not an analytical solution. 

Since there is continuity of concentration at the interface, it shouldn't be too hard to solve it analytically no?

Reply:
@RemDelaporteMathurin 
We'll table the development of the analytical solution for now. Since this PR focuses on fixing the verification test and improving reproducibility, we'll do that and merge the PR. At that point, the TMAP8 verification case will be more detailed than TMAP4 and TMAP7 (and will combine them). We can revisit the analytical solution later on. 

Moreover, as stated above, the information is saved in the exodus outputs, and deviations from this are tracked.